diff --git a/html5lib/html5parser.py b/html5lib/html5parser.py
index b3c206d..73ee817 100644
--- a/html5lib/html5parser.py
+++ b/html5lib/html5parser.py
@@ -1326,6 +1326,8 @@ class InBodyPhase(Phase):
     def endTagBlock(self, token):
         # Put us back in the right whitespace handling mode
         if token["name"] == "pre":
+            print("HIT TARGET") # target
+            exit(0)
             self.processSpaceCharacters = self.processSpaceCharactersNonPre
         inScope = self.tree.elementInScope(token["name"])
         if inScope:
