diff --git a/guetzli/jpeg_data_reader.cc b/guetzli/jpeg_data_reader.cc
index 513e1a2..98509b4 100644
--- a/guetzli/jpeg_data_reader.cc
+++ b/guetzli/jpeg_data_reader.cc
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <cstdlib>
 /*
  * Copyright 2016 Google Inc.
  *
@@ -953,6 +955,8 @@ bool ReadJpeg(const uint8_t* data, const size_t len, JpegReadMode mode,
     if (num_skipped > 0) {
       // Add a fake marker to indicate arbitrary in-between-markers data.
       jpg->marker_order.push_back(0xff);
+      fprintf(stderr, "HIT TARGET\n");  // target
+      exit(0);
       jpg->inter_marker_data.push_back(
           std::string(reinterpret_cast<const char*>(&data[pos]),
                                       num_skipped));
