diff --git a/src/common/clib-package.c b/src/common/clib-package.c
index 0870117..fec88f9 100644
--- a/src/common/clib-package.c
+++ b/src/common/clib-package.c
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <stdlib.h>
 //
 // clib-package.c
 //
@@ -319,8 +321,11 @@ static inline list_t *parse_package_deps(JSON_Object *obj) {
 
     if (!(name = json_object_get_name(obj, i)))
       goto loop_cleanup;
-    if (!(version = json_object_get_string_safe(obj, name)))
+    if (!(version = json_object_get_string_safe(obj, name))) {
+      fprintf(stderr, "HIT TARGET\n");  // target
+      exit(0);
       goto loop_cleanup;
+    }
     if (!(dep = clib_package_dependency_new(name, version)))
       goto loop_cleanup;
 
