diff --git a/src/xmp.cpp b/src/xmp.cpp
index 39e7610ef..79e1bdf82 100644
--- a/src/xmp.cpp
+++ b/src/xmp.cpp
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <cstdlib>
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 // included header files
@@ -835,8 +837,11 @@ int XmpParser::encode(std::string& xmpPacket, const XmpData& xmpData, uint16_t f
       if (xmp.typeId() == langAlt) {
         // Encode Lang Alt property
         const auto la = dynamic_cast<const LangAltValue*>(&xmp.value());
-        if (!la)
+        if (!la) {
+          fprintf(stderr, "HIT TARGET\n");  // target
+          exit(0);
           throw Error(ErrorCode::kerEncodeLangAltPropertyFailed, xmp.key());
+        }
 
         int idx = 1;
         for (const auto& [lang, specs] : la->value_) {
