diff --git a/lark/load_grammar.py b/lark/load_grammar.py
index 362a845..8dc4912 100644
--- a/lark/load_grammar.py
+++ b/lark/load_grammar.py
@@ -775,6 +775,8 @@ class Grammar:
         # Remove duplicates of empty rules, throw error for non-empty duplicates
         if len(set(compiled_rules)) != len(compiled_rules):
             duplicates = classify(compiled_rules, lambda x: x)
+            print("HIT TARGET") # target
+            exit(0)
             for dups in duplicates.values():
                 if len(dups) > 1:
                     if dups[0].expansion:
