diff --git a/lark/load_grammar.py b/lark/load_grammar.py
index 362a845..5621918 100644
--- a/lark/load_grammar.py
+++ b/lark/load_grammar.py
@@ -884,6 +884,8 @@ def resolve_term_references(term_dict):
                         raise GrammarError("Terminal used but not defined: %s" % item.name)
                     assert term_value is not None
                     exp.children[0] = term_value
+                    print("HIT TARGET") # target
+                    exit(0)
                     changed = True
                 else:
                     assert isinstance(item, Tree)
