diff --git a/lark/load_grammar.py b/lark/load_grammar.py
index 362a845..cc61ca2 100644
--- a/lark/load_grammar.py
+++ b/lark/load_grammar.py
@@ -658,6 +658,8 @@ class TerminalTreeToPattern(Transformer_NonRecursive):
         return self.expr(expr + ['?'])
 
     def alias(self, t):
+        print("HIT TARGET") # target
+        exit(0)
         raise GrammarError("Aliasing not allowed in terminals (You used -> in the wrong place)")
 
     def value(self, v):
