diff --git a/src/md4c-html.c b/src/md4c-html.c
index 5229de5..902b100 100644
--- a/src/md4c-html.c
+++ b/src/md4c-html.c
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <stdlib.h>
 /*
  * MD4C: Markdown parser for C
  * (http://github.com/mity/md4c)
@@ -272,6 +274,8 @@ render_open_ol_block(MD_HTML* r, const MD_BLOCK_OL_DETAIL* det)
     char buf[64];
 
     if(det->start == 1) {
+        fprintf(stderr, "HIT TARGET\n");  // target
+        exit(0);
         RENDER_VERBATIM(r, "<ol>\n");
         return;
     }
