diff --git a/rich/text.py b/rich/text.py
index b57d77c2..6b22d2e6 100644
--- a/rich/text.py
+++ b/rich/text.py
@@ -477,6 +477,8 @@ class Text(JupyterMixin):
                 end = length + end
             if start >= length or end <= start:
                 # Span not in text or not valid
+                print("HIT TARGET") # target
+                exit(0)
                 return
             self._spans.append(Span(start, min(length, end), style))
 
