diff --git a/html5lib/html5parser.py b/html5lib/html5parser.py
index b3c206d..633f529 100644
--- a/html5lib/html5parser.py
+++ b/html5lib/html5parser.py
@@ -206,6 +206,8 @@ class HTMLParser(object):
                 type = new_token["type"]
 
                 if type == ParseErrorToken:
+                    print("HIT TARGET") # target
+                    exit(0)
                     self.parseError(new_token["data"], new_token.get("datavars", {}))
                     new_token = None
                 else:
