diff --git a/fsspec/asyn.py b/fsspec/asyn.py
index a040efc..72e9b4e 100644
--- a/fsspec/asyn.py
+++ b/fsspec/asyn.py
@@ -927,6 +927,8 @@ def mirror_sync_methods(obj):
 
     for method in async_methods + dir(AsyncFileSystem):
         if not method.startswith("_"):
+            print("HIT TARGET") # target
+            exit(0)
             continue
         smethod = method[1:]
         if private.match(method):
