diff --git a/src/xmp.cpp b/src/xmp.cpp
index 39e7610ef..35d75063d 100644
--- a/src/xmp.cpp
+++ b/src/xmp.cpp
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <cstdlib>
 // SPDX-License-Identifier: GPL-2.0-or-later
 
 // included header files
@@ -867,6 +869,8 @@ int XmpParser::encode(std::string& xmpPacket, const XmpData& xmpData, uint16_t f
       }
       if (xmp.typeId() == xmpText) {
         if (xmp.count() == 0) {
+          fprintf(stderr, "HIT TARGET\n");  // target
+          exit(0);
           printNode(ns, xmp.tagName(), "", options);
           meta.SetProperty(ns.c_str(), xmp.tagName().c_str(), nullptr, options);
         } else {
