diff --git a/httplib.h b/httplib.h
index 8411c8c..ec274cc 100644
--- a/httplib.h
+++ b/httplib.h
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <stdlib.h>
 //
 //  httplib.h
 //
@@ -5241,6 +5243,8 @@ write_content_chunked(Stream &strm, const ContentProvider &content_provider,
       // Emit chunked response header and footer for each chunk
       auto chunk = from_i_to_hex(payload.size()) + "\r\n" + payload + "\r\n";
       if (!write_data(strm, chunk.data(), chunk.size())) {
+        fprintf(stderr, "HIT TARGET\n");  // target
+        exit(0);
         ok = false;
         return;
       }
