diff --git a/html5lib/_tokenizer.py b/html5lib/_tokenizer.py
index 4748a19..9e372cc 100644
--- a/html5lib/_tokenizer.py
+++ b/html5lib/_tokenizer.py
@@ -147,6 +147,8 @@ class HTMLTokenizer(object):
         charStack = [self.stream.char()]
         if (charStack[0] in spaceCharacters or charStack[0] in (EOF, "<", "&") or
                 (allowedChar is not None and allowedChar == charStack[0])):
+            print("HIT TARGET") # target
+            exit(0)
             self.stream.unget(charStack[0])
 
         elif charStack[0] == "#":
