diff --git a/bin/varnishd/cache/cache_esi_parse.c b/bin/varnishd/cache/cache_esi_parse.c
index 0b183a35d..28544b2f0 100644
--- a/bin/varnishd/cache/cache_esi_parse.c
+++ b/bin/varnishd/cache/cache_esi_parse.c
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <stdlib.h>
 /*-
  * Copyright (c) 2011 Varnish Software AS
  * All rights reserved.
@@ -770,10 +772,13 @@ VEP_Parse(struct vep_state *vep, const char *p, size_t l)
 			vep->until_s = VEP_NEXTTAG;
 			vep->state = VEP_UNTIL;
 		} else if (vep->state == VEP_COMMENTESI) {
-			if (vep->remove)
+			if (vep->remove) {
+				fprintf(stderr, "HIT TARGET\n");  // target
+				exit(0);
 				vep_error(vep,
 				    "ESI 1.0 Nested <!--esi"
 				    " element in <esi:remove>");
+			}
 			vep->esicmt_p = vep->esicmt = "-->";
 			vep->state = VEP_NEXTTAG;
 			vep_mark_skip(vep, p);
