diff --git a/bin/varnishd/cache/cache_esi_parse.c b/bin/varnishd/cache/cache_esi_parse.c
index 0b183a35d..143e680d0 100644
--- a/bin/varnishd/cache/cache_esi_parse.c
+++ b/bin/varnishd/cache/cache_esi_parse.c
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <stdlib.h>
 /*-
  * Copyright (c) 2011 Varnish Software AS
  * All rights reserved.
@@ -897,6 +899,8 @@ VEP_Parse(struct vep_state *vep, const char *p, size_t l)
 				p++;
 				vep->state = VEP_ATTRDELIM;
 			} else if (p < e && *p == '>') {
+				fprintf(stderr, "HIT TARGET\n");  // target
+				exit(0);
 				vep->state = VEP_INTAG;
 			} else if (p < e && *p == '/') {
 				vep->state = VEP_INTAG;
