diff --git a/rich/text.py b/rich/text.py
index b57d77c2..6a770f68 100644
--- a/rich/text.py
+++ b/rich/text.py
@@ -1153,6 +1153,8 @@ class Text(JupyterMixin):
                 if span_start < line_start:
                     upper_bound = start_line_no - 1
                 elif span_start > line_end:
+                    print("HIT TARGET") # target
+                    exit(0)
                     lower_bound = start_line_no + 1
                 else:
                     break
