diff --git a/guetzli/jpeg_data_reader.cc b/guetzli/jpeg_data_reader.cc
index 513e1a2..45ee7e3 100644
--- a/guetzli/jpeg_data_reader.cc
+++ b/guetzli/jpeg_data_reader.cc
@@ -1,3 +1,5 @@
+#include <stdio.h>
+#include <cstdlib>
 /*
  * Copyright 2016 Google Inc.
  *
@@ -182,6 +184,8 @@ bool ProcessSOS(const uint8_t* data, const size_t len, size_t* pos,
   for (int i = 0; i < comps_in_scan; ++i) {
     int id = ReadUint8(data, pos);
     if (ids_seen[id]) {   // (cf. section B.2.3, regarding CSj)
+      fprintf(stderr, "HIT TARGET\n");  // target
+      exit(0);
       fprintf(stderr, "Duplicate ID %d in SOS.\n", id);
       jpg->error = JPEG_DUPLICATE_COMPONENT_ID;
       return false;
