diff --git a/html5lib/html5parser.py b/html5lib/html5parser.py
index b3c206d..4e3bc34 100644
--- a/html5lib/html5parser.py
+++ b/html5lib/html5parser.py
@@ -1246,6 +1246,8 @@ class InBodyPhase(Phase):
         if self.tree.elementInScope("ruby"):
             self.tree.generateImpliedEndTags()
             if self.tree.openElements[-1].name != "ruby":
+                print("HIT TARGET") # target
+                exit(0)
                 self.parser.parseError()
         self.tree.insertElement(token)
 
